compass/landice/tests/ismip6_run/ismip6_ais_proj2300/set_up_experiment.py

```diff
@@ -0,0 +1,254 @@
+                     f"{tf_search_path}: {fcgFileList}")
+
+        # copy calving mask files for exp11-14
+        useCalvingMask = False
+        if exp_fcg[-2:].isdigit():
+            exp_num = int(exp_fcg[-2:])
+            if exp_num >= 11:
+                mask_search_path = os.path.join(
+                    forcing_basepath, exp_fcg,
+                    'Antarctica_8to30km_ice_shelf_collapse_mask_*.nc')
+                fcgFileList = glob.glob(mask_search_path)
+                if len(fcgFileList) == 1:
+                    mask_path = fcgFileList[0]
+                    mask_fname = os.path.split(mask_path)[-1]
+                    shutil.copy(mask_path, self.work_dir)
+                    useCalvingMask = True
+                else:
+                    sys.exit("ERROR: Did not find exactly one matching "
+                             "calving mask file at "
+                             f"{mask_search_path}: {fcgFileList}")
+
+        # Make stream modifications based on files that were determined above
+        stream_replacements = {'input_file_SMB_forcing': smb_fname,
+                               'input_file_TF_forcing': tf_fname}
+        if self.exp == 'hist':
+            stream_replacements['input_file_init_cond'] = init_cond_fname
+            stream_replacements['input_file_region_mask'] = region_mask_fname
+            stream_replacements['input_file_melt_params'] = melt_params_fname
+        else:
+            stream_replacements['input_file_init_cond'] = \
+                'USE_RESTART_FILE_INSTEAD'
+            stream_replacements['input_file_region_mask'] = \
+                'USE_RESTART_FILE_INSTEAD'
+            stream_replacements['input_file_melt_params'] = \
+                'USE_RESTART_FILE_INSTEAD'
+        if self.exp in ['hist', 'ctrlAE']:
+            stream_replacements['forcing_interval'] = 'initial_only'
+        else:
+            stream_replacements['forcing_interval'] = '0001-00-00_00:00:00'
+
+        self.add_streams_file(
+            resource_location,
+            'streams.landice.template',
+            out_name='streams.landice',
+            template_replacements=stream_replacements)
+
+        if useCalvingMask:
+            mask_stream_replacements = {'input_file_calving_mask_forcing_name':
+                                        mask_fname}
+            self.add_streams_file(
+                resource_location, 'streams.mask_calving',
+                out_name='streams.landice',
+                template_replacements=mask_stream_replacements)
+
+        if use_vM_calving:
+            vM_param_path = section.get('von_mises_parameter_path')
+            vM_stream_replacements = {'input_file_VM_params': vM_param_path}
+            self.add_streams_file(
+                resource_location, 'streams.vM_params',
+                out_name='streams.landice',
+                template_replacements=vM_stream_replacements)
+
+        # Set up namelist and customize as needed
```

Comment:
I don't know if you want to deal with this here, but I used these for the 4km runs:
```
config_front_mass_bal_grounded = 'ismip6'
config_use_3d_thermal_forcing_for_face_melt = .true.
```

Reply:
I created a new cfg option for including facemelting or not.